tests/unit/test_download.py

```diff
@@ -51,7 +51,27 @@ def _fake_session_get(*args, **kwargs):
 
 
 def test_user_agent():
-    PipSession().headers["User-Agent"].startswith("pip/%s" % pip.__version__)
+    user_agent = PipSession().headers["User-Agent"]
+
+    assert user_agent.startswith("pip/%s" % pip.__version__)
+
+
+@pytest.mark.parametrize('name, expected_like_ci', [
+    ('BUILD', False),
```

Comment:
maybe this too should be constructed from that array

Reply:
I will make some changes, but we want to be careful not to make the test depend on the code it is testing. If someone removes something from `CI_ENVIRONMENT_VARIABLES`, we want the test to fail rather than automatically pass.